charts/topolvm/templates/node/daemonset.yaml

```diff
@@ -98,13 +98,16 @@ spec:
             - /csi-node-driver-registrar
             - --csi-address={{ .Values.node.kubeletWorkDirectory }}/plugins/{{ include "topolvm.pluginName" . }}/node/csi-topolvm.sock
             - --kubelet-registration-path={{ .Values.node.kubeletWorkDirectory }}/plugins/{{ include "topolvm.pluginName" . }}/node/csi-topolvm.sock
+            - --health-port=9809
```

Comment:
`--health-port` is deprecated. Please use `--http-endpoint` instead.
https://github.com/kubernetes-csi/node-driver-registrar/tree/release-2.5#optional-arguments